greymatter/modules/fabric/tasks.tf

```diff
@@ -5,7 +5,7 @@ resource "aws_ecs_task_definition" "control-api" {
   family                   = "control-api"
   container_definitions    = local.control_api_container
   requires_compatibilities = ["EC2"]
-  network_mode             = "bridge"
+  network_mode             = "awsvpc"
```

Comment:
Factor this network_mode into a variable with a default value of "awsvpc"

Reply:
If this is configurable we'll run into issues with the ecs-services discovery set up - only awsvpc works with dns record type "A".  - from [aws docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html):

```
If the task definition that your service task specifies uses the bridge or host network mode, an SRV record is the only supported DNS record type. Create an SRV record for each service task. The SRV record must specify a container name and container port combination from the task definition.
```

Reply:
That's true. Let's leave it for now. We'd have to figure out how to make the other SD strategy configurable as well.